### PR TITLE
Addressed review comments on heat driver

### PR DESCRIPTION
**Pull request by YogeshRajmane**
Addressed review comments on heat driver

@ashutosh-mishra @njagadish @mak-454

### CODE REVIEW

File: gbpservice/nfp/orchestrator/config_drivers/heat_driver.py [hunk 1]
```diff
@@ -307,9 +327,9 @@ def _create_policy_target_for_vip(self, auth_token,
             if lb_pool_ids and lb_pool_ids[0]['vip_id']:
                 lb_vip = self.neutron_client.get_vip(
                     auth_token, lb_pool_ids[0]['vip_id'])
-                self._create_pt(auth_token, provider_tenant_id, provider['id'],
-                                "service_target_vip_pt",
-                                port_id=lb_vip['vip']['port_id'])
+                self.gbp_client.create_policy_target(
+                    auth_token, provider_tenant_id, provider['id'],
+                    "service_target_vip_pt", lb_vip['vip']['port_id'])
```
Comment (ashutosh-mishra) [inline gbpservice/nfp/orchestrator/config_drivers/heat_driver.py]: can you append vip port_id in name to distinguish multiple vip ports

File: gbpservice/nfp/orchestrator/config_drivers/heat_driver.py [hunk 2]
```diff
@@ -421,7 +441,9 @@ def _get_consumers_for_chain(self, auth_token, provider):
                     break
 
         if not redirect_prs:
-            raise
+            LOG.ERROR(_LE("Redirect rule doesn't exist in policy target rule "
```
Comment (ashutosh-mishra) [inline gbpservice/nfp/orchestrator/config_drivers/heat_driver.py]: error in caps

File: gbpservice/nfp/orchestrator/config_drivers/heat_driver.py [hunk 3]
```diff
@@ -1157,16 +1231,26 @@ def _update(self, auth_token, resource_owner_tenant_id, service_profile,
...
-                    heatclient, stack_id, 'update')
+                try:
+                    heatclient.update(stack_id, stack_template, stack_params)
+                except Exception as err:
+                    msg = ('Node update failed. There can be a chance if the '
+                           'service is LOADBALANCER, the related '
+                           'configuration would have been lost. Please check '
+                           'with the ADMIN for issue of failure and '
+                           're-initiate the update node once again.')
+                    LOG.exception(_LE('%(msg) NODE-ID: %(node_id)s '
```
Comment (ashutosh-mishra) [inline gbpservice/nfp/orchestrator/config_drivers/heat_driver.py]: should be %(msg)s

File: gbpservice/nfp/orchestrator/config_drivers/heat_driver.py [hunk 4]
```diff
@@ -1181,33 +1265,12 @@ def _update(self, auth_token, resource_owner_tenant_id, service_profile,
...
-                               'instance_id': service_chain_instance['id'],
-                               'tenant_id': provider_tenant_id,
-                               'err': str(err)})
-
-                raise NodeUpdateException(
-                    node=service_chain_node['id'],
-                    instance_id=service_chain_instance['id'],
-                    tenant=provider_tenant_id, msg=msg)
-
+                LOG.exception(_LE('%(msg)') % {'msg': msg})
```
Comment (ashutosh-mishra) [inline gbpservice/nfp/orchestrator/config_drivers/heat_driver.py]: should be %(msg)s

File: gbpservice/nfp/orchestrator/config_drivers/heat_driver.py [hunk 5]
```diff
@@ -703,6 +743,12 @@ def _update_node_config(self, auth_token, tenant_id, service_profile,
                 stack_template[resources_key][fw_key][properties_key][
                     'description'] = str(firewall_desc)
         elif service_type == pconst.VPN:
+            rvpn_l3_policy = self._get_rvpn_l3_policy(auth_token,
+                provider, update)
```
Comment (njagadish) [inline gbpservice/nfp/orchestrator/config_drivers/heat_driver.py]: This is not needed. We are not supporting remote VPN. Lets remove this.

**Pull request merged**